test/multi_request.js

```diff
@@ -0,0 +1,55 @@
+const IdToAddressBiMap = artifacts.require("IdToAddressBiMap")
+const SnappBaseCore = artifacts.require("SnappBaseCore")
+const MultiRequest = artifacts.require("MultiRequest")
+const MintableERC20 = artifacts.require("./ERC20Mintable.sol")
+
+const { setupEnvironment } = require("./utilities.js")
+
+contract("MultiRequest", async (accounts) => {
+  const [token_owner, user_1] = accounts
+
+  beforeEach(async () => {
+    const lib1 = await IdToAddressBiMap.new()
+    
+    await SnappBaseCore.link(IdToAddressBiMap, lib1.address)
+    const lib2 = await SnappBaseCore.new()
+
+    await MultiRequest.link(IdToAddressBiMap, lib1.address)
+    await MultiRequest.link(SnappBaseCore, lib2.address)
+  })
+
+  describe("Test full request batches", () => {
```

Comment:
Should this rather be part of the existing SnappBase tests? I think it logically fits in there and it feels kind of weird to just have a file for this part of the tests.

Reply:
Works for me! Will change.

Reply:
My only concern here, is the contract linking in the "Before Each".